resources/lang/pl/beatmapsets.php

```diff
@@ -50,13 +50,14 @@
             ],
             'favourite' => 'Dodaj tę beatmapę do ulubionych',
             'unfavourite' => 'Usuń tę beatmapę z ulubionych',
+            'favourited_count' => '+ 1 inna!|+ :count inne!|+ :count innych!',
         ],
         'stats' => [
             'cs' => 'Wielkość kółek',
             'cs-mania' => 'Ilość klawiszy',
             'drain' => 'Spadek HP',
             'accuracy' => 'Celność',
```

Comment:
`Celność` makes it see more like aim-related, how about `Dokładność` or something?

Reply:
myślałem o `Precyzja` raczej

Reply:
nawet lepiej